src/TodoList.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const todoList = [
```

Comment:
I would name them `items` as those represent items and not the todo list in general